### PR TITLE
Modify all-in-one benchmark

### PR DESCRIPTION
**Pull request by pengyb2001**
## Description

Update 8192 prompt in all-in-one
Add cpu_embedding param for linux api

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -403,19 +404,19 @@ def run_transformer_int4_gpu(repo_id,
...
         model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True,
-                                                     use_cache=True).eval()
+                                                     use_cache=True, cpu_embedding=cpu_embedding).eval()
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
     else:
         if "4bit" in repo_id:
             model = AutoModelForCausalLM.load_low_bit(model_path, optimize_model=True,
-                                            trust_remote_code=True, use_cache=True).eval()
+                                            trust_remote_code=True, use_cache=True, cpu_embedding=cpu_embedding).eval()
         else:
```
Comment (hkvision) [inline python/llm/dev/benchmark/all-in-one/run.py:420]: missing cpu_embedding in this branch?

Comment (pengyb2001) [reply python/llm/dev/benchmark/all-in-one/run.py:420]: updated in this branch.

File: python/llm/dev/benchmark/all-in-one/config.yaml [hunk 2]
```diff
@@ -32,5 +32,5 @@ test_api:
   # - "bigdl_ipex_int8"
   # - "speculative_cpu"
   # - "deepspeed_transformer_int4_cpu" # on Intel SPR Server
-cpu_embedding: False # whether put embedding to CPU (only avaiable now for gpu win related test_api)
+cpu_embedding: False # whether put embedding to CPU
```
Comment (hkvision) [inline python/llm/dev/benchmark/all-in-one/config.yaml:35]: also modify this in the readme?

**Review by hkvision — APPROVED** [review]

File: python/llm/dev/benchmark/all-in-one/config.yaml (hunk 2 above)
Comment (pengyb2001) [reply python/llm/dev/benchmark/all-in-one/config.yaml:35]: updated, ready to merge now

**Pull request merged**